frontend/zc_messaging/src/component/ThreadList/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import ThreadItem from "./ThreadItem"
+import "./threadList.css"
+
+const ThreadList = ({ threadListData }) => (
+  <div className="threadList-Wrap">
+    {threadListData.map((thread, index) => (
+      <ThreadItem key={index} Thread={thread} />
```

Comment:
an index is not a key

Reply:
Changed to the unique id of each thread message